### DISCUSSION TITLE
Viewer role

### DISCUSSION THREAD

**Discussion by PatrikFomin** [uxbox/uxbox#4267] [Q&A]
Is there a way to enable a "Viewer role" that would be able to access a team or project but not actually edit anything?
We have setup SSO using Google and the best would be if all members could just go to the self-hosted URL, sign-in and then an admin is assigning them a "View" only role to the team or projects they should see. Right now I only see Admin or Editor but I don't want people to be able to edit anything, just View it.

**Comment by madalenapmelo-kp** [comment]
Hi @PatrikFomin,

Thanks for the suggestion! We already have this in feature planned into our backlog, you can keep up with the progress [here](https://tree.taiga.io/project/penpot/us/1056?backlog-q=viewer&no-milestone=1) or by keeping an eye on issue #3793.

I'm closing this discussion but please feel free to share any other suggestions on [our Community](https://community.penpot.app/) as this forum has been discontinued. Thanks 🙂
